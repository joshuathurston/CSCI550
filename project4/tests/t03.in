NOWITSOHAPPENEDTHATONONEO
CCASIONTHEPRINCESSSGOLDEN
BALLDIDNOTFALLINTOTHELITT
LEHANDWHICHSHEWASHOLDINGU
PFORIT
