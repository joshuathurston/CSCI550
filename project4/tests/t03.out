106 51 61 78 8 90 27 17 50 26 25 39 84 47 54 95 56 14 80 13 76 70 48 11 23 34 40 88 60 101 44 98 7 16 77 33 69 87 82 92 85 83 55 37 96 64 29 104 3 72 46 53 94 75 63 71 52 62 49 38 79 12 22 97 20 57 0 31 65 24 6 45 93 21 19 30 102 58 67 1 10 100 9 35 36 103 43 86 91 28 5 42 41 105 59 15 32 68 74 18 66 4 73 99 89 81 2 
